NEWS.md

```diff
@@ -1,3 +1,7 @@
+# epireview 1.1.2
+
+* Add filter_df_for_metaprop function
+
```

Comment:
Same as above, I think this needs to be 1.1.3 as previous PR with 1.1.2 already exists